x-pack/plugins/beats_management/public/pages/beat/detail.tsx

```diff
@@ -30,11 +29,11 @@ export const BeatDetailPage = (props: BeatDetailPageProps) => {
     return <div>Beat not found</div>;
   }
   const configurationBlocks = flatten(
-    beat.full_tags.map((tag: ClientSideBeatTag) => {
-      return tag.configurations.map(configuration => ({
+    beat.full_tags.map((tag: BeatTag) => {
+      return tag.configuration_blocks.map(configuration => ({
         ...configuration,
         // @ts-ignore one of the types on ClientSideConfigurationBlock doesn't define a "module" property
```

Comment:
This comment should be edited or removed since the code it's commenting on has changed.